packages/url_launcher/url_launcher_ios/pigeons/messages.dart

```diff
@@ -27,7 +27,7 @@ enum InAppLoadResult {
   /// The URL was successfully loaded.
   success,
 
-  /// The URL did not load successfully.
+  /// The URL did not load successfully (or close the SFSafariViewController earlier).
```

Comment:
What do you think of something like this?

```suggestion
  /// The URL did not load successfully (or the user closed the view controller before it loaded).
```

Reply:
I have made the changes. Thank you for your suggestion.